packages/support/lib/image-util.js

```diff
@@ -22,18 +22,25 @@ async function getJimpImage(data) {
     if (!_.isString(data) && !_.isBuffer(data)) {
       return reject(new Error('Must initialize jimp object with string or buffer'));
     }
-    // if data is a string, assume it is a base64-encoded image
-    if (_.isString(data)) {
-      data = Buffer.from(data, 'base64');
-    }
+
+    const imagePayload = _.isBuffer(data)
+      ? data
+      // if data is a string, assume it is a base64-encoded image
+      : Buffer.from(String(data), 'base64');
+    const truncatedData = _.truncate(
+      _.isBuffer(data) ? data.toString('utf8') : data,
```

Comment:
`_.isBuffer(data)` isn't type-safe; it returns a `boolean` but does not narrow the type of `data` to `Buffer`.  `Buffer.isBuffer` does, however. 

```suggestion
      Buffer.isBuffer(data) ? data.toString('utf8') : data,
```

Reply:
fixed